main.go

```diff
@@ -116,25 +121,28 @@ func main() {
 	fmt.Printf("Message hash: 0x%s\n", hex.EncodeToString(messageHash))
 
 	if signerErr != nil {
+		if address {
+			log.Fatalf("Error creating signer: %v", signerErr)
+		}
```

Comment:
nit: can remove this

Reply:
fixed: https://github.com/base/eip712sign/pull/18/commits/c6881c0a240e8cf59ec26b9eea5ddb5607657c35